internal/graph/check.go

```diff
@@ -848,6 +848,268 @@ func (c *LocalChecker) checkComputedUserset(_ context.Context, req *ResolveCheck
 	}
 }
 
+// checkTTUSlowPath is the slow path for checkTTU where we cannot short-circuit TTU evaluation and
+// resort to dispatch check on its children.
+func (c *LocalChecker) checkTTUSlowPath(ctx context.Context, req *ResolveCheckRequest, rewrite *openfgav1.Userset, iter storage.TupleKeyIterator) (*ResolveCheckResponse, error) {
+	ctx, span := tracer.Start(ctx, "checkTTUSlowPath")
+	defer span.End()
+
+	var errs error
+	var handlers []CheckHandlerFunc
+
+	typesys, ok := typesystem.TypesystemFromContext(ctx)
+	if !ok {
+		return nil, fmt.Errorf("typesystem missing in context")
+	}
+
+	computedRelation := rewrite.GetTupleToUserset().GetComputedUserset().GetRelation()
```

Comment:
This could potentially be multiple relations. Consider unions and relations that have both a directly assignable and computed relation.

Reply:
Unions are evaluated at a different level at the current time, and computed is currently not supported. So for the context of this implementation, only one needs to be handled. This is, in-fact the _existing_ implementation of in main.

Reply:
@willvedd re: "This could potentially be multiple relations" - if the TTU is `define viewer1: viewer2 from parent`, the computed relation will be `viewer2` and it will always be 1